pkg/k8s/secrets/transformers_test.go

```diff
@@ -0,0 +1,108 @@
+package secrets
+
+import (
+	"testing"
+
+	"github.com/SAP/stewardci-core/pkg/k8s/fake"
+	"gotest.tools/assert"
+	v1 "k8s.io/api/core/v1"
+)
+
+func Test_UniqueNameTransformer(t *testing.T) {
+	secret := fake.SecretWithType("name", "secret", v1.SecretTypeDockercfg)
```

Comment:
Sample values should be easily recognizable as such. I prefer the pattern of adding a number to the base name, e.g. `name1` or `secret1`.